src/ansys/fluent/core/session.py

```diff
@@ -47,8 +47,11 @@ def _parse_server_info_file(file_name: str):
 
 def _get_datamodel_attributes(session, attribute: str):
     try:
-        preferences_module = importlib.import_module(
-            f"ansys.fluent.core.generated.datamodel_{session._version}." + attribute
```

Comment:
The hardcoded module path is removed. Now, module will be loaded from dynamic `CODEGEN_OUTDIR`.